python/dgl/transform/module.py

```diff
@@ -14,11 +14,21 @@
 #   limitations under the License.
 #
 """Modules for transform"""
+# pylint: disable= no-member, arguments-differ, invalid-name
+
+from scipy.linalg import expm
 
 from .. import convert
 from .. import backend as F
+from .. import function as fn
 from . import functional
 
+try:
+    import torch
+    from torch.distributions import Bernoulli
+except ImportError:
+    pass
```

Comment:
Is there a version requirement?

Reply:
The version requirement should be handled by [this](https://github.com/dmlc/dgl/blob/master/python/dgl/backend/pytorch/tensor.py#L18).

Reply:
I see. It seems that some of the modules are PyTorch only. How about import torch in the init function?